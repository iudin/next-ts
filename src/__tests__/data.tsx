import * as React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Data from '../components/data';

describe('data component', () => {
  const component = shallow(<Data data={undefined} />);
  it('renders properly', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
  it('renders empty data message when no data provided', () => {
    expect(component.find('styled.p').text()).toEqual('No data!');
  });
});
