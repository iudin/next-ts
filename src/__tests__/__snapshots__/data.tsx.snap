// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data component renders properly 1`] = `
<styled.p>
  No data!
</styled.p>
`;
